<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortWidget</class>
 <widget class="QWidget" name="SortWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>440</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>主菜单</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="worksnumRadioButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>作品数量</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="averagerateRadioButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>80</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>作品平均分</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sortButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>排 序</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>511</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>571</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>要对所有用户信息进行排序并显示，请选择排序标准：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
